js/index.js

```diff
@@ -56,23 +56,20 @@ for (let i = 0; i < skills.length; i++){
     event.target.reset();
 });
 
-//handles the response from the server
-let githubRequest = new XMLHttpRequest();
-    githubRequest.open('GET','https://api.github.com/users/dianayossi/repos');
-    githubRequest.send();
-
-    githubRequest.addEventListener('load', () => {
-        let repositories = JSON.parse(githubRequest.response);
+// Fetch API
+fetch('https://api.github.com/users/dianayossi/repos')
+    .then(response => response.json())
+    .then((data) => {
+        let repositories = data; //storing data in repositories variable 
```

Comment:
You can just name your argument `repositories` instead, no need to have it saved twice